detox/src/devices/drivers/android/exec/ADB.js

```diff
@@ -297,22 +289,4 @@ class ADB {
   }
 }
 
-function assertEmulatorHasPort(device, stdout) {
-  if (device.port) {
-    return;
-  }
-
-  const errorMessage = [
-    `Failed to determine telnet port for emulator device '${device.adbName}'!`,
-    `Please help us out by reporting dump below in: https://github.com/wix/Detox/issues/1427`,
-    `------ BEGIN DUMP ------`,
-    `adb devices base64: ${encodeBase64(stdout)}`,
-    `adb devices: ${stdout}`,
-    `port: ${device.port}`,
-    `------  END DUMP  ------`,
-  ].join('\n');
-
-  throw new Error(errorMessage);
-}
-
```

Comment:
@rotemmiz @noomorph Friends, I think it's been long enough.

Reply:
Agree.